api/src/test/java/org/pmiops/workbench/api/CloudTaskInitialCreditsExhaustionControllerTest.java

```diff
@@ -590,12 +589,13 @@ public void handleInitialCreditsExhaustionBatch_singleAlert_forExhaustedAndByoBi
 
   @Test
   public void
-      handleInitialCreditsExhaustionBatch_disableFreeTierWorkspacesOnly_whenUserHasMultipleWorkspaces()
+      handleInitialCreditsExhaustionBatch_disableInitialCreditsWorkspacesOnly_whenUserHasMultipleWorkspaces()
           throws MessagingException {
-    workbenchConfig.billing.defaultFreeCreditsDollarLimit = 100.0;
+    workbenchConfig.billing.defaultInitialCreditsDollarLimit = 100.0;
 
     final DbUser user = createUser(SINGLE_WORKSPACE_TEST_USER);
-    final DbWorkspace freeTierWorkspace = createWorkspace(user, SINGLE_WORKSPACE_TEST_PROJECT);
+    final DbWorkspace InitialCreditsWorkspace =
```

Comment:
Minorest of nits: Could you make the first i lowercase?
initialCreditsWorkspace